lib/cmd/fh3/services/data-sources.js

```diff
@@ -0,0 +1,39 @@
+/* globals i18n */
+var common = require("../../../common");
+var fhc = require("../../../fhc");
+var fhreq = require("../../../utils/request");
+
+module.exports = {
+  'desc' : i18n._('List service data-sources'),
+  'examples' :
+  [{
+    cmd : 'fhc services data-sources --service=<service>',
+    desc : i18n._('List all data-sources of the <service>')
+  }],
+  'demand' : ['service'],
+  'alias' : {
+    'service':'s',
+    'json' : 'j',
+    0 : 'service',
+    1 : 'json'
+  },
+  'describe' : {
+    'service' : i18n._('Unique 24 character GUID of the service'),
+    'json' : i18n._('Output into json format')
+  },
+  'customCmd': function(params, cb) {
+    fhc.services.read({service:params.service}, function(err, service) {
+      if (err) {
+        return cb(err);
+      }
+
+      common.doGetApiCall(fhreq.getFeedHenryUrl(), "api/v2/services/" + service.apps[0].guid + "/data_sources", "Error reading Service data sources. ", function(err, dataSources) {
+        if (err) {
+          return cb(err);
+        }
+        params._table = common.createTableForDataSources(dataSources || [], service);
```

Comment:
this behaviour is different from the previous implementation I think. Should it only print out in the table format when the `--table` flag is passed?